docs/introduction/Sidechain-SDK-extension.rst

```diff
@@ -404,7 +402,7 @@ Inside those functions, all required action could be defined, and with them also
 
   4. Add response classes
 
-As a result of an API request, the result shall be sent back via HTTP response. In a typical case, we could have two different responses: operation is successful, or some error had appeared during processing the API request. SDK provides following way to declare those API responses:
+As a result of an API request, the result shall be sent back via HTTP response. In a typical case, we could have two different responses: operation is successful, or some error had appeared during processing the API request. The SDK provides following way to declare those API responses:
```

Comment:
@zennermac 

The result will be sent back via HTTP response as an outcome of an API request. In a most cases, we could have two different responses: operation is successful, or an error had appeared during processing the API request. The SDK provides the following way to declare those API responses: